From 2b5835f56084267bb8addcd7d771f77029a62230 Mon Sep 17 00:00:00 2001
From: Roman Stratiienko <r.stratiienko@gmail.com>
Date: Mon, 21 Oct 2024 01:41:03 +0300
Subject: [PATCH 3/3] common: Remove libdav1d

Android-15 (And LOS21) has it embedded into the framework code already.

Signed-off-by: Roman Stratiienko <r.stratiienko@gmail.com>
---
 common/codecs/board.mk | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/common/codecs/board.mk b/common/codecs/board.mk
index fed5d9b..7881392 100644
--- a/common/codecs/board.mk
+++ b/common/codecs/board.mk
@@ -6,9 +6,6 @@
 
 BCDC_PATH := $(patsubst $(CURDIR)/%,%,$(dir $(abspath $(lastword $(MAKEFILE_LIST)))))
 
-BOARD_BUILD_AOSPEXT_DAV1D := true
-BOARD_DAV1D_SRC_DIR := glodroid/vendor/dav1d
-
 BOARD_BUILD_AOSPEXT_FFMPEG := true
 BOARD_FFMPEG_SRC_DIR := glodroid/vendor/ffmpeg
 
-- 
2.43.0

